examples/tutorial/tutorial_notebooks/introduction/omnisci_on_native/exercise_1.ipynb

```diff
@@ -0,0 +1,227 @@
+   "source": [
+    "Modin is a drop-in replacement for pandas that distributes the computation \n",
+    "across all of the cores in your machine or in a cluster.\n",
+    "In practical terms, this means that you can continue using the same pandas scripts\n",
+    "as before and expect the behavior and results to be the same. The only thing that needs\n",
+    "to change is the import statement. Normally, you would change:\n",
+    "\n",
+    "```python\n",
+    "import pandas as pd\n",
+    "```\n",
+    "\n",
+    "to:\n",
+    "\n",
+    "```python\n",
+    "import modin.pandas as pd\n",
+    "```\n",
+    "\n",
+    "Changing this line of code will allow you to use all of the cores in your machine to do computation on your data. One of the major performance bottlenecks of pandas is that it only uses a single core for any given computation. Modin exposes an API that is identical to Pandas, allowing you to continue interacting with your data as you would with Pandas. There are no additional commands required to use Modin locally. Partitioning, scheduling, data transfer, and other related concerns are all handled by Modin under the hood."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Exercise 1: How to use Modin\n",
+    "\n",
+    "**GOAL**: Learn how to import Modin to accelerate and scale pandas workflows."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<p style=\"text-align:left;\">\n",
+    "        <h1>pandas on a multicore laptop\n",
+    "    <span style=\"float:right;\">\n",
+    "        Modin on a multicore laptop\n",
+    "    </span>\n",
+    "\n",
+    "<div>\n",
+    "<img align=\"left\" src=\"../img/pandas_multicore.png\"><img src=\"../img/modin_multicore.png\">\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Concept for exercise: Dataframe constructor\n",
+    "\n",
+    "Often when playing around in Pandas, it is useful to create a DataFrame with the constructor. That is where we will start.\n",
+    "\n",
+    "```python\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**10, 2**5))\n",
+    "df = pd.DataFrame(frame_data)\n",
+    "```\n",
+    "\n",
+    "When creating a dataframe from a non-distributed object, it will take extra time to partition the data. When this is happening, you will see this message:\n",
+    "\n",
+    "```\n",
+    "UserWarning: Distributing <class 'numpy.ndarray'> object. This may take some time.\n",
+    "```\n",
+    "\n",
+    "Multithreading is implemented using Ray, OmniSci or Dask. On default using Ray. Let's run examples on the OmniSci backend. For this, we need to activate OmniSci in the Modin config.\n",
+    "\n",
+    "**In a later exercise, we will discuss times when it is not possible to speed up the computation, even with multiprocessing or multithreading.**"
```

Comment:
```suggestion
    "Modin uses Ray as an execution engine by default. Since this notebook is related to OmniSci, let's run examples on the OmniSci engine. For reaching this, we need to activate OmniSci either via Modin config or Modin environment variable. See more in [OmniSci usage](put a link to the page here) section.\n",
```

Reply:
@YarShev this was changed